docs/software/cloud.md

```diff
@@ -110,6 +110,80 @@ Nextcloud gives you access to all your files wherever you are. Store your docume
     ncc [followed by the desired command]
     ```
 
+=== "Nextcloud 'Brute force protection'"
+
+    Nextcloud offers built in brute force protection and additionally a plugin ***Brute-force settings***.  
+    This will delay your login rate in case of several failed login attempts.
+
+    This protection can be extended with Fail2Ban (see following tab).
+
+    See also: <https://apps.nextcloud.com/apps/bruteforcesettings>
+
+=== "Fail2Ban integration"
+
+    Using Fail2Ban your can block users after failed login attempts. This hardens your system, e.g. against brute force attacks.
+
+    - Set options in the ***Nextcloud configuration file*** (typical `/var/www/nextcloud/config/config.php`):
+
+        - Add trusted domains if not already set via the `'trusted_domains'` entry.
+
+            ```ini
+            'trusted_domains' =>
+             array (
+               0 => 'localhost',
+               1 => '<your_trusted_domain>',
+             ),
+            ```
+
+        - Check resp. edit the log file options.  
+          Set the `logtimezone` to the value of your system.  
+          Check whether the `logfile` path matches the entry done in the *Fail2Ban jail file* (`nextcloud.local`, see below).
+
+            ```ini
+            'log_type' => 'file',
+            'logtimezone' => 'Europe/Berlin',
+            'logfile' => '/var/log/nextcloud.log',
```

Comment:
On DietPi, the default Nextcloud log file location is (as Nextcloud default is `<data dir>/nextcloud.log`):
```
/mnt/dietpi_userdata/nextcloud_data/nextcloud.log
```
It makes sense to not move it to `/var/log` since this is by default RAMlog and cleared every hour, which breaks mid/long term client banning.

I would skip the log section here completely, as everything defaults to reasonable values and use `/mnt/dietpi_userdata/nextcloud_data/nextcloud.log` (respectively `<data dir>/nextcloud.log` with customised data dir set via `SOFTWARE_NEXTCLOUD_DATADIR` `dietpi.txt` setting.

This section is very long, hence my aim to not add too much info which is not relevant for default DietPi installs 😉.

Reply:
OK, removed these points and mentioned the defaults.